Directory.Build.props

```diff
@@ -48,7 +48,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
     <PackageProjectUrl>https://github.com/Microsoft/dotnet-apiport</PackageProjectUrl>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```

Comment:
Were you going to set it to true, or remove?

Reply:
I trie to remove it, but the default value seems false. That's why I set it to true instead.